packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ScmMethodProviderCollection.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Microsoft.Generator.CSharp.ClientModel.Providers
 {
-    internal class ScmMethodProviderCollection : MethodProviderCollection
+    public class ScmMethodProviderCollection : MethodProviderCollection
```

Comment:
it does not look like any functionality changed, do we need to override this?

Reply:
Yes, we are overriding the methods here.